MatrixSDK/Data/MXRoom.m

```diff
@@ -1020,7 +1020,7 @@ - (MXHTTPOperation*)sendImage:(NSData*)imageData
         kMXMessageBodyKey: filename,
         @"url": fakeMediaURI,
         @"info": [@{
-            @"mimetype": mimetype,
+            @"mimetype": (mimeType ?: @"@application/octet-stream"),
```

Comment:
Hmm what's the reason for the `@` before the mime type?

Reply:
Incorrect NSString marker. Fixed